src/pages/addCard/components/CardPasswordInput.tsx

```diff
@@ -0,0 +1,45 @@
+import { RefObject } from "react";
+import Label from "../../../components/input/Label";
+import Input from "../../../components/input/Input";
```

Comment:
```suggestion
import { Label, Input } from "../../../components";
```
처럼 불러올 수 있도록 묶어주는 것도 괜찮을 것 같네요!